src/views/RecIM/components/List/Listing/index.js

```diff
@@ -77,60 +83,100 @@ const SeriesListing = ({ series }) => {
 };
 
 const ActivityListing = ({ activity }) => {
-  let registrationStart = DateTime.fromISO(activity.RegistrationStart);
+  const [activityType, setActivityType] = useState();
+  const [currentCapacity, setCurrentCapacity] = useState(
+    <span style={{ display: 'inline-block' }}>
```

Comment:
Interesting use of a span, but if it is necessary here in order to display the loader inline then it is ok.